crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -262,6 +291,7 @@ impl InboundGroupSession {
             backed_up: AtomicBool::from(pickle.backed_up).into(),
             algorithm: pickle.algorithm.into(),
             imported: pickle.imported,
+            safety: if pickle.trusted { KeySafety::Safe } else { KeySafety::Unsafe },
```

Comment:
If we just use the `claimed_trust` flag, then this conversion becomes unnecessary as well.